test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -518,6 +519,9 @@ class PTLTestRunner(Plugin):
     def __init__(self):
         Plugin.__init__(self)
         self.param = None
+        self.repeat_tests = 1
+        self.test_repeat = 1
```

Comment:
You can rename self.test_repeat name to something like self.default_test_run_count.

Reply:
@shilpakodli I think self.test_run_count is also ok.